File: platform-backend/src/com/wwf/shrimp/application/models/OrganizationStage.java
package com.wwf.shrimp.application.models;

/**
 * @author user
 *
 */
public class OrganizationStage extends LookupEntity {
	
	private int orderIndex;
	private long orgID;
	private String colorHexCode;
	
	public int getOrderIndex() {
		return orderIndex;
	}
	public void setOrderIndex(int orderIndex) {
		this.orderIndex = orderIndex;
	}
	public long getOrgID() {
		return orgID;
	}
	public void setOrgID(long orgID) {
		this.orgID = orgID;
	}
	public String getColorHexCode() {
		return colorHexCode;
	}
	public void setColorHexCode(String colorHexCode) {
		this.colorHexCode = colorHexCode;
	}

}

File: platform-desktop/src/app/models/OrganizationStage.ts
import { LookupEntity } from './lookupEntity';

export class OrganizationStage extends LookupEntity {
    orderIndex: number;
    orgID: number;
    colorHexCode: string='';

    public static clone(stage:OrganizationStage){
        let clone: OrganizationStage = new OrganizationStage();
        clone.id = stage.id;
        clone.name = stage.name;
        clone.value = stage.value;
        clone.colorHexCode = stage.colorHexCode;
        clone.orderIndex = stage.orderIndex;
        clone.orgID = stage.orgID;

        return clone;

   }
}